app/controllers/navigation/Navigator.scala

```diff
@@ -41,6 +43,17 @@ class Navigator @Inject()(appConfig: AppConfig, auditService: AuditService) {
       case _ => Results.Redirect(call)
     }
 
+  def continueTo(mode: Mode, factory: Mode => Call)(implicit req: JourneyRequest[AnyContent], hc: HeaderCarrier): Result =
+    redirectTo(mode.next, factory)
```

Comment:
Why we need `continueTo` if it's just a wrapper for `redirectTo`?

Reply:
`mode.next` - redirect preserve mode - continue change it